TSOClient/tso.client/UI/Panels/UILotControl.cs

```diff
@@ -280,6 +281,11 @@ void vm_OnDialog(FSO.SimAntics.Model.VMDialogInfo info)
 
             VMDialogType type = (info.Operand == null) ? VMDialogType.Message : info.Operand.Type;
 
+            if (info.Operand.MessageStringID == 19)
```

Comment:
This is a hack. You must edit phoneglobals.iff with volcanic and change the dialog primitive type to this instead. (and include the final result as a piff)

Reply:
Removed this hack and will include an updated `jobcontroller.piff` which changes the Operand Type for this Dialog to "FSOJob" /  127 per the enum.